core/primitives-core/src/version.rs

```diff
@@ -211,6 +210,7 @@ impl ProtocolFeature {
             | ProtocolFeature::RemoveAccountWithLongStorageKey => 68,
             ProtocolFeature::StatelessValidation => 69,
             ProtocolFeature::EthImplicitAccounts => 70,
+            ProtocolFeature::BLS12381 => 71,
```

Comment:
We want `BLS12381` feature to be released together with `StatelessValidation`. Similarly as protocol v69 contains several features.

Reply:
fixed: https://github.com/near/nearcore/pull/11818/commits/ea0400f21f86194ddcd9ac630335c7abb59cbb46